console/src/views/Multimedia.js

```diff
@@ -34,16 +31,6 @@ const IFrame = kind({
 	}
 });
 
-const ListItemOverlay = kind({
```

Comment:
ImageItem doesn't have something similar to selectionOverlay, so this had to be deleted